shivyc/tree/nodes.py

```diff
@@ -668,6 +668,10 @@ def extract_params(self, decl):
 
         for param in func_decl.args:
             decl_info = self.get_decl_infos(param)[0]
+            # check if void is the function argument,we skip
```

Comment:
The patch (IMHO, as an outsider to the project) looks pretty clean, the only suggestion is with this code comment. Would be nice to reword/reformat it a bit, e.g. "check if "void" is the function argument, and skip it".